topics/admin/tutorials/customization-manual/tutorial.md

```diff
@@ -0,0 +1,167 @@
+This tutorial will teach you three basic customizations you can make to Galaxy:
+
+  - Setting a brand text
+  - Adding a custom welcome page
+  - Customizing the masthead using themes
+
+For this tutorial we will be setting up a fictional instance called "Galaxy Mars".
+Feel free to use the included material in the hands-on sections, or provide your own.
+
+> <agenda-title></agenda-title>
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+## Custom Brand
+
+The brand text in Galaxy refers to the text you can see in the masthead of some Galaxy instances.
+This text will appear in the masthead, as well as the sites title.
+It is an easy way to set your instance apart, and make it more identifiable.
+
+![screenshot of the start page of Galaxy Europe](images/galaxy-europe.png "Galaxy Europe uses the brand text \"Europe\"")
+
+> <hands-on-title>Setting a brand text</hands-on-title>
+> 
+> 1. Open your galaxy config file `config/galaxy.yml` and add the following option under `galaxy:`
+> 
+>    ```yaml
+>    brand: Mars
+>    ```
+>
+> 2. Start your galaxy server using `sh run.sh`
+>
+> 3. Navigate to `https://localhost:8080`
+>
+>    ![screenshot of fictional "Galaxy Mars" start page, with the brand text set to "Mars"](images/galaxy-mars-brand.png "Your Galaxy start page should now look like this")
+{: .hands_on}
+
+## Custom Welcome Page
+
+The welcome page is an html document embedded in Galaxy's start page.
+It is what users see when first navigating to your instance.
+This page can be used to communicate what your instance is about, and share news and updates with your users.
+
+> <hands-on-title>Creating a custom welcome page</hands-on-title>
+>
+> 0. (optional) Set the location of your welcome page in `galaxy.yml`
+> 
+>    Under `galaxy:` add the following option:
+>
+>    ```yaml
+>    welcome_url: /static/your-welcome-page.html
+>    ```
+>
+>    Not setting this will cause the url to default to `/static/welcome.html`
+>
+> 1. Open `/static/welcome.html`, or create a new html file at the location set in Step 0
+>
+>    It is advisable to keep a copy of the default `welcome.html`, as it can serve as a useful reference.
+>
+> 2. Add some Content
+>
+>    Let's add a title and some text to our welcome page:
+>
+>    ```html
+>    <!DOCTYPE html>
+>    <html lang="en">
+>        <head>
+>            <meta charset="utf-8">
+>        </head>
+>        <body>
+>            <h1>Welcome to the Galaxy Mars instance!</h1>
+>            <p>The only Galaxy instance on mars.</p>
+>        </body>
+>    </html>
+>    ```
+>
+>    This works the same as any other html page. You can add styles in the head, or import some scripts.
+>    
+>    Make sure your welcome page has exactly one `h1` element, which describes the page.
+>    This will act as the heading for your start-page, which can help assistive technologies.
+{: .hands_on}
+
+todo: add note about style/font
```

Comment:
Bonus todo: could mention that it's convenient to make this an iframe like we do on .*

Reply:
Resolved